.circleci/config.yml

```diff
@@ -597,6 +597,7 @@ workflows:
                 - stage
                 - prod
                 - USAGOV-1083-dev-merge-point
+                - USAGOV-1448-www-host-for-tome-sync
```

Comment:
Please remove the feature branches from the CircleCI config.

Reply:
Removed